Update Scheduled Views index page (#4826)
SumoLogic/sumologic-documentation: docs/manage/scheduled-views/index.md

* Scheduled View queries run once per minute.
* Queries that run against Scheduled Views return search results much faster because the data is pre-aggregated before the query is run. 
* Creating a Scheduled View for a query can vastly reduce the amount of data scanned at search time.

> * If you do not have access to specific data because of Role-Based Access Control (RBAC) or role filter restrictions, it’s important to note that the Scheduled Views you create will not reflect that data. Ensuring proper access is crucial for comprehensive insights.

Author: ```suggestion
* If your Scheduled View do not have (or modified later to not have) access to specific data, because of Role-Based Access Control (RBAC) or role filter restrictions, it’s important to note that the Scheduled Views will not reflect that data. Ensuring proper access is crucial for comprehensive insights.
```
